test/common/scale_up_replicas_threescale.go

```diff
@@ -0,0 +1,152 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	"testing"
+	"time"
+
+	threescalev1 "github.com/3scale/3scale-operator/pkg/apis/apps/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	numberOfReplicas int64 = 2
+	scaleUpReplicas  int64 = 3
+	name                   = "3scale"
+	namespace              = "redhat-rhmi-3scale"
+)
+
+func TestReplicasInThreescale(t *testing.T, ctx *TestingContext) {
+
+	createWorkspaceRetryInterval := time.Second * 20
+	createWorkspaceTimeout := time.Minute * 7
```

Comment:
```suggestion
	retryInterval := time.Second * 20
	timeout := time.Minute * 7
```
Just a name change here. A workspace is a CodeReady workspace concept